prowler/providers/aws/services/opensearch/opensearch_service.py

```diff
@@ -91,12 +91,16 @@ def __describe_domain__(self, regional_clients):
                     DomainName=domain.name
                 )
                 domain.arn = describe_domain["DomainStatus"]["ARN"]
-                domain.endpoint_vpc = None
+                domain.vpc_endpoints = None
                 if "Endpoints" in describe_domain["DomainStatus"]:
                     if "vpc" in describe_domain["DomainStatus"]["Endpoints"]:
-                        domain.endpoint_vpc = describe_domain["DomainStatus"][
-                            "Endpoints"
-                        ]["vpc"]
+                        domain.vpc_endpoints = [
+                            vpc
+                            for vpc in describe_domain["DomainStatus"][
+                                "Endpoints"
+                            ].values()
+                        ]
+
```

Comment:
What is the reason behind this?

Reply:
If we are sure that this is not being used in the checks please remove it. Double check this with @sergargar please.

Reply:
I confirm that `endpoint_vpc` is not used.